setupext.py

```diff
@@ -812,14 +797,7 @@ def check(self):
         except ImportError:
             msgs += [bad_pytest]
 
-        if PY3min:
-            msgs += ['using unittest.mock']
-        else:
-            try:
-                import mock
-                msgs += ['using mock %s' % mock.__version__]
-            except ImportError:
-                msgs += [msg_template.format(package='mock')]
+        msgs += ['using unittest.mock']
```

Comment:
would not even bother with the message, it's just a boring regular stdlb module now